cypress/integration/core-opensearch-dashboards/opensearch-dashboards/datasource-management-plugin/5_version_decouple.spec.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
+  describe('Create datasources', () => {
+    before(() => {
+      // Clean up before creating new data sources for testing
+      cy.deleteAllDataSources();
+      cy.createDataSourceNoAuthWithTitle('ds_1');
+      cy.wait(6000);
+    });
+
+    after(() => {
+      // Clean up after all test are run
+      cy.deleteAllDataSources();
+      // remove the default data source
+      cy.setAdvancedSetting({
+        defaultDataSource: '',
+      });
+    });
+
+    describe('Check datasource contains version decouple related information', () => {
+      it('check installed plugins and data source version information is showed', () => {
+        cy.request({
+          method: 'GET',
+          url: '/api/saved_objects/_find?type=data-source',
```

Comment:
Because the version decouple can only be triggered through adding datasource using UI. The createDataSourceNoAuthWithTitle in the before hook would help to add datasource using UI page.  It has not been enabled through API yet.